### ISSUE TITLE
did u ever done the overfitting check on openchat v1.0

### ISSUE THREAD

**Issue by renatz** [imoneoi/openchat#28]
although your file has split the dataset into the training set and validation set, it seems that training for 5 epochs with 6k samples will encounter overfitting.

**Comment by imoneoi** [comment]
According to the LIMA paper, the evaluation loss is not correlated with the final human evaluation. Thus, in the context of LLM fine-tuning, "overfitting" may either improve or degrade performance.

**Comment by renatz** [comment]
> According to the LIMA paper, the evaluation loss is not correlated with the final human evaluation. Thus, in the context of LLM fine-tuning, "overfitting" may either improve or degrade performance.

Oh, thanks for your information. 
Can I ask if your group just takes the last checkpoint for inference, or also use ChatGPT to score the effect of each checkpoint and decide which one to use for inference?

**Comment by imoneoi** [comment]
We used the last checkpoint for inference. We have run GPT-4 scoring on different sets of training epochs/hyperparameters etc. to select the best one.

**Comment by imoneoi** [comment]
Closing this issue now. If you have further questions, plz re-open.
